### PR TITLE
CLDR-16181 TZ database 2022g updates, accurate Mexican exemplar city …

### PR DESCRIPTION
**Pull request by yumaoka**
…names for Mexican zones

CLDR-16181

- [x] This PR completes the ticket.

### CODE REVIEW

**Review by macchiati — CHANGES_REQUESTED** [review]
To allow Juarez to be voted on in v43, you will need to also modify the regex in:

SubmissionLocales.PATHS_ALLOWED_IN_LIMITED

**Comment by yumaoka** [comment]
@macchiati 
> To allow Juarez to be voted on in v43, you will need to also modify the regex in:
> 
> SubmissionLocales.PATHS_ALLOWED_IN_LIMITED

I don't know whether my change done in the code is correct, because I don't know about this part much. Please confirm.

File: common/bcp47/timezone.xml [hunk 1]
```diff
@@ -280,6 +280,7 @@ For terms of use, see http://www.unicode.org/copyright.html
             <type name="mwblz" description="Blantyre, Malawi" alias="Africa/Blantyre"/>
             <type name="mxchi" description="Chihuahua, Mexico" alias="America/Chihuahua"/>
             <type name="mxcun" description="Cancún, Mexico" alias="America/Cancun"/>
+            <type name="mxcjs" description="Ciudad Juárez, Mexico" alias="America/Ciudad_Juarez"/>
```
Comment (markusicu) [inline common/bcp47/timezone.xml:283]:
```suggestion
            <type name="mxcjz" description="Ciudad Juárez, Mexico" alias="America/Ciudad_Juarez"/>
```
There is no 's' in the name of the city.

**Review by markusicu — COMMENTED** [review]
one change please

FYI @gnrunge

File: common/bcp47/timezone.xml (hunk 1 above)
Comment (yumaoka) [reply common/bcp47/timezone.xml:283]: @markusicu We use LOCODE by default - https://service.unece.org/trade/locode/mx.htm

MX  CJS | Ciudad Juárez | Ciudad Juarez | CHH | ---4---- | AI | 0212 |   | 3144N 10629W
-- | -- | -- | -- | -- | -- | -- | -- | --

So "mxcjs" is correct.

Comment (markusicu) [reply common/bcp47/timezone.xml:283]: ok tnx never mind... but then the LOCODE is wrong :-P

**Review by markusicu — APPROVED** [review]
lgtm

**Review by macchiati — APPROVED** [review]
Looks good. Please verify with smoke test (or cldr staging) that a vetter can vote on the item before closing the ticket.

**Pull request merged**